accounts/abi/bind/base.go

```diff
@@ -149,6 +149,25 @@ func DeployContract(opts *TransactOpts, abi abi.ABI, bytecode []byte, backend Co
 	return c.address, tx, c, nil
 }
 
+// Address returns the contract deployment address.
+func (c *BoundContract) Address() common.Address {
+	return c.address
+}
+
+// GetABI returns the contract ABI Object of the BoundContract.
+func (c *BoundContract) GetABI() abi.ABI {
+	return c.abi
+}
```

Comment:
Triage discussion: why do you need these two accessors? In order to create a bound contract, you already had both of them. In general: who would ever need to obtain it _from_ a bound contract?

Reply:
The ABI is already generated as a global variable, so you can just access "package.MyContractABI"

Reply:
That's fair enough on the `common.Address` front. The motivation behind exposing it in the bound contract was to have access to the ABI from the bound contract. It looks a lot cleaner to call from the bound contract. Instead of calling `package.MyContractABI` just call `boundContractInstance.GetABI`. I personally think it's a cleaner abstraction, but if it's egregious enough to block the PR then I'll rip it out.

Reply:
Also `package.MyContractABI` was supposed to be "Deprecated" though it's still available in the codebase. You should access it as `package.MyContract.ABI` now. However, this is really gross with upgradable contracts or contracts with longer names. Obviously, you can alias, but if you have the bound contract from the generated code that's already been parsed, it makes sense to have an accessor of some sort. Again, this isn't a hill I'm willing to die on, but I think is a nice convenience.